Aflores plymouth (#234)
osisoft/Edge-Data-Store-Docs: content/overview/opc-ua-quick-start.md

1. Save the file to the device with EDS installed using a file name based on the adapter instance name. For example, to use the adapter instance created during installation, which is `OpcUa1`, name the file `OpcUa1Datasource.json`.

> 1. Run the following curl script from the directory where the file is located, updating the file name and the destination in the script if needed.

Reviewer: ```suggestion
1. Run the following `curl` script from the directory where the file is located and update the file name and the destination in the script if needed.
```

## Configure OPC UA data selection

> When you create the data source file, the OPC UA adapter auto generates the data selection file, which lists all available streams in the designated data source. To configure the data selection file:

Reviewer: ```suggestion
When you create the data source file, the OPC UA adapter auto generates the data selection file and lists all available streams in the designated data source. To configure the data selection file:
```

       }
   ]
   ```

> 2. To ingress a stream to Edge Data Store, change the value of the **Selected** key from `false` to `true`. All streams in the auto generated data selection file are initially set to `false`.

Reviewer: ```suggestion
1. To ingress a stream to Edge Data Store, change the value of the `Selected` key from `false` to `true`. All streams in the auto generated data selection file are initially set to `false`.
```

> 3. Save the the file.
> 
> 4. Run the following curl script from the directory where the file is located, updating the file name and destination in the script if needed:

Reviewer: ```suggestion
1. Save the the file.

1. Run the following `curl ` script from the directory where the file is located and update the file name and destination in the script if needed:
```

> 4. Run the following curl script from the directory where the file is located, updating the file name and destination in the script if needed:

Author: ```suggestion
1. Run the following curl script from the directory where the file is located, updating the file name and destination in the script if needed:
```
